j5_zoloto/backends.py

```diff
@@ -30,7 +30,11 @@ class ZolotoSingleHardwareBackend(MarkerCameraInterface, Backend):
     def discover(cls) -> Set[Board]:
         """Discover boards that this backend can control."""
         return {
-            ZolotoCameraBoard("0", cls(0)),  # Choose the first camera only.
+            ZolotoCameraBoard(
+                str(camera_id),
+                cls(camera_id)
+            )
+            for camera_id in find_camera_ids()
```

Comment:
As this now supports multiple cameras, please could you update the name of the class? It is currently `ZolotoSingleHardwareBackend`, which is wrong following this change.

The docstring of the class will need to be updated too.